src/com/projectkorra/projectkorra/BendingPlayer.java

```diff
@@ -110,6 +110,8 @@ public void addCooldown(Ability ability) {
 	 * @param cooldown The cooldown time
 	 */
 	public void addCooldown(String ability, long cooldown) {
+		if(cooldown <= 0)

```

Comment:
We prefer that you follow one true brace style and add the braces on single-line conditional statements like this.